ccf/app/views.py

```diff
@@ -1,6 +1,17 @@
-from django.shortcuts import redirect
+from django.contrib.auth.decorators import login_required
```

Comment:
Are all of these imports used...? I only see `UserCreationForm`, `render`, and `reverse_lazy`

Reply:
`redirect` is no longer used, but all the other ones are. Good catch.